libportal/portal-private.h

```diff
@@ -53,7 +53,7 @@ struct _XdpPortal {
   guint screencast_interface_version;
 };
 
-#define PORTAL_BUS_NAME "org.freedesktop.portal.Desktop"
+extern gchar PORTAL_BUS_NAME[];
```

Comment:
I have a particular distaste for `extern` - maybe you could do something like `#define PORTAL_BUS_NAME (portal_get_bus_name ())` instead?

Reply:
done!